### PR TITLE
[MIG] shopinvader_locomotive_wishlist: Migration to 14.0

### PR DESCRIPTION
**Pull request by chafique-delli**
- [ ] Depends on https://github.com/shopinvader/odoo-shopinvader/pull/859

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Comment by rousseldenis** [comment]
@chafique-delli Could you rebase ?

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1 @@
-# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+sale-workflow
```
Comment (simahawk) [inline oca_dependencies.txt:1]: not needed

File: test-requirements.txt [hunk 2]
```diff
@@ -9,3 +9,13 @@ openupgradelib @ git+https://github.com/OCA/openupgradelib.git
 
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
+
```
Comment (simahawk) [inline test-requirements.txt:12]: drop these changes

**Review by simahawk — CHANGES_REQUESTED** [review]
please include this very important fix from v13 https://github.com/shopinvader/odoo-shopinvader/commit/4f4d4e6fdf43aa5b3688cf6db7fb024c6b9c1a23

**Comment by simahawk** [comment]
Replaced by #975 

Sorry guys, is getting insane w/ pending merges sweat_smile and I kind of need to move faster wink

I've included the fix from v13 mentioned above.

**Pull request closed without merging**